ctr/src/flavors/ctr128.rs

```diff
@@ -18,6 +18,13 @@ pub struct CtrNonce128<N: ArrayLength<u128>> {
     nonce: GenericArray<u128, N>,
 }
 
+impl<N: ArrayLength<u128>> core::fmt::Debug for CtrNonce128<N> {
+    #[inline]
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
```

Comment:
Usually, I prefer to import `core::fmt` and then use `fmt::Debug`, `fmt::Formatter`, and `Fmt::Result`. But this code should be fine as well.